app/controllers/placements/organisations_controller.rb

```diff
@@ -1,6 +1,22 @@
 class Placements::OrganisationsController < ApplicationController
-  def index
-    @schools = current_user.schools
-    @providers = current_user.providers
+  before_action :redirect_to_organisation_if_only_one, only: :index
+  def index; end
+
+  private
+
+  def redirect_to_organisation_if_only_one
+    set_memberships
+
+    if @memberships.one?
+      redirect_to placements_organisation_path(organisation)
+    end
+  end
+
+  def set_memberships
+    @memberships = current_user.memberships.filter(&:placements_service).sort_by(&:name)
```

Comment:
We'll be able to do this more elegantly when we have done some additional work on scoping (tickets have been written and we're working on those today)